moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1827,6 +1827,8 @@ as the new values that correspond to the group */
   /** \brief This function is only called in debug mode */
   bool checkCollisionTransforms() const;
 
+  static std::string name_;
```

Comment:
This seems to be superfluos. Either use `RobotState::name_` or this `static const name_` everywhere.
Personally, I would prefer the latter actually.

Reply:
`name_` has to be static to be used by static methods like `testJointSpaceJump` and static member variables have to be initialized outside the class definition. I don't know of a cleaner way to do this. If you have a cleaner syntax please add your commit on top of mine. 

https://stackoverflow.com/questions/1563897/c-static-constant-string-class-member